src/app_service/service/chat/service.nim

```diff
@@ -348,3 +348,88 @@ QtObject:
       let errDesription = e.msg
       error "error: ", errDesription
       return
+
+  method addGroupMembers*(self: Service, chatId: string, pubKeys: seq[string]) =
+    try: 
+      let response = status_chat.addGroupMembers(chatId, pubKeys)
+      self.emitUpdate(response)
```

Comment:
@jrainville response used here. Did i missed something?

Reply:
My bad, I don't know why I missed it 🤔